README.md

```diff
@@ -0,0 +1,73 @@
+# dnsprove.js 
```

Comment:
Could you add a more detailed API description to the README. I'd like to be able to read the API or at least reference it as I read your example to make sure I understand what each part returns. Something like this: https://github.com/jefflau/jest-fetch-mock#api